src/main/java/sisyphus/Storage.java

```diff
@@ -0,0 +1,114 @@
+package sisyphus;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+import sisyphus.task.Deadline;
+import sisyphus.task.Event;
+import sisyphus.task.TaskList;
+import sisyphus.task.ToDo;
+
+
+/**
+ * Storage class that stores and reads a local data file in the user's computer.
+ */
+public class Storage {
+    private String currentDirectory = System.getProperty("user.dir");
```

Comment:
I think it will be good to give some javadocs for your fields (even though its private haha)